portfolio/src/main/java/com/google/sps/servlets/UserData.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps.servlets;
+
+class UserData {
+    private boolean isLoggedIn = false;
```

Comment:
I'd make all of these fields `private final ...`

Reply:
Done.